common_components/orchestration/modules/kubernetes/outputs.tf

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+output "gcp_service_account_email" {
```

Comment:
Why this output? I can see it being useful, just curious as it doesn't seem to be used.

Reply:
we can most likely remove - at some point I thought it would have been useful to output that to fix the permission issue with the GCP SA reading GCS, but there may be a simpler way to do that